extension/src/popups/MinervaMenu.html

```diff
@@ -16,72 +16,102 @@
-                aria-selected="true">You</a>
-            </li>
-            <li class="nav-item">
-              <a class="nav-link" id="ranking-tab" data-toggle="tab" href="#nav-ranking" role="tab" aria-controls="ranking"
-                aria-selected="false">Ranking</a>
-            </li>
-            <li class="nav-item">
-              <a class="nav-link" id="store-tab" data-toggle="tab" href="#nav-store" role="tab" aria-controls="store"
-                aria-selected="false">Store</a>
-            </li>
-          </ul>
-          <div class="tab-content" id="nav-tabContent">
-            <div class="tab-pane fade show active overflow-auto" id="nav-you" role="tabpanel" aria-labelledby="nav-you-tab">
-              <div class="card">
-                <div class="card-body">
-                  <h5 class="card-title" id="studentName-youPanel" ></h5>
-                  <p class="card-text">Total minutes watched: 69:42</p>
+        <div class="container">
+          <div class="row">
+            <div class="col">
+              <div class="card-body" id="side-bar-owned-badges"></div>
+            </div>
+          </div>
+          <div class="row">
+            <div class="col">
+              <div class="card card-body">
+                <ul class="nav nav-pills justify-content-center">
+                  <li class="nav-item">
+                    <a class="nav-link active" id="you-tab" data-toggle="tab" href="#nav-you" role="tab" aria-controls="you"
+                      aria-selected="true">You</a>
+                  </li>
+                  <li class="nav-item">
+                    <a class="nav-link" id="ranking-tab" data-toggle="tab" href="#nav-ranking" role="tab" aria-controls="ranking"
+                      aria-selected="false">Ranking</a>
+                  </li>
+                  <li class="nav-item">
+                    <a class="nav-link" id="store-tab" data-toggle="tab" href="#nav-store" role="tab" aria-controls="store"
+                      aria-selected="false">Store</a>
+                  </li>
+                </ul>
+                <div class="tab-content" id="nav-tabContent">
+                  <div class="tab-pane fade show active overflow-auto" id="nav-you" role="tabpanel" aria-labelledby="nav-you-tab">
+                    <div class="card">
+                      <h5 class="card-title" id="studentName-youPanel"></h5>
+                      <div class="container">
+                        <div class="row">
+                          <div class="col">
+                            <img class="icon" src="images/clock.png" width="15" height="15" title="Time Watched">
+                            <p class="font-weight-light" id="time-record"></p>
+                          </div>
+                          <div class="col">
+                            <img class="icon" src="images/coin.png" width="15" height="15" data-toggle="tooltip" title="Coin Balance">
```

Comment:
Does `data-toggle="tooltip` do anything for the current `img` tag? If I am missing some context here, then `Time Watched` should also have this?

Reply:
removed